File: database/sql/tables/wallets.rb
module Tables
  class Wallets < SQLAlchemy::Entities::Table
    def initialize
      super(:wallets)

      add_column(:ids, :uuid, primary: true, default: :uuid_generate_v4)
      add_column(:name, :string, unique: true)
      add_column(:currency, :string)

      timestamps(:created_at, :updated_at)
    end

    def self.name
      :wallets
    end
  end
end

File: mister_krabz/entities/wallets.py
from mister_krabz.entities.entity import Entity


class Wallet(Entity):
    def __init__(self, name, id=None, created_at=None, updated_at=None):
        super().__init__(id=id, created_at=created_at, updated_at=updated_at)

        self.name = name
